poller-lambdas/src/pollers/ap/apPoller.ts

```diff
@@ -191,6 +192,11 @@ function itemWithContentToDesiredOutput({
 			abstract,
 			originalContentText: originalXmlContent,
 			ednote,
+			associations: associations
```

Comment:
I'd be tempted to call this field `imageIds`, because at the moment we don't have a more general schema for representing associations so I think it makes sense to only store image ids in this field. What do you think?

Assuming we do that, we'd also want to filter this list for `.type === 'picture'`